packages/data/addon/adapters/bard-facts.ts

```diff
@@ -122,12 +124,12 @@ export default class BardFactsAdapter extends EmberObject {
    * @param {function} aliasFunction function that returns metrics from aliases
    * @return {String} sort param value
    */
-  _buildSortParam(request, aliasFunction = a => a) {
+  _buildSortParam(request: RequestV1, aliasFunction: AliasFn = a => a): string | undefined {
     const { sort } = request;
 
     if (sort && sort.length) {
       return sort
-        .map(sortMetric => {
+        .map((sortMetric: { metric: string; direction: string }) => {
```

Comment:
Should we have a sortDirection type?
`type sortDirection = 'asc' | 'desc';`

Reply:
👍 Slowly been putting the v2 types into v1. We have a `SortDirection` type.